src/index.html

```diff
@@ -15,8 +15,106 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a
+        href="index.html"
+        class="logo__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav__list">
+          <li>
+            <a
+              class="nav__link nav__link__first is-active"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link"
+              href="index.html"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li>
+            <a
+              class="nav__link nav__link__active__hover"
```

Comment:
The class name 'nav__link__active__hover' is not very meaningful. Consider using class names that represent the content or purpose rather than styles or tag names.